storage/aptosdb/src/lib.rs

```diff
@@ -1569,6 +1560,9 @@ impl DbWriter for AptosDB {
                 num_transaction_infos
             );
 
+            // TODO(joshlind): include confirm_or_save_frozen_subtrees in the change set
+            // bundle below.
```

Comment:
Thinking about it, crashing after this should not make the DB in a unusable state, right? (at least we can likely make it so, because we don't determine the progress by reading that table, we read the JMT roots and txns instead).

Reply:
Awesome, we'll that's good news 😄 Thanks @msmouse!